### PR TITLE
Wired Dashboard to App via React Route

### PR DESCRIPTION
**Pull request by vunderkind**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by aapetsi — APPROVED** [review]
All good

**Review by shaunorpen — APPROVED** [review]
Not much to review here, but I can see the component and it appears to work, so approving!

File: src/components/DashboardHome/DashContent.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Tabs } from 'antd';
+import './Dashboard.css';
+import Pane1 from './Pane1';
+import './Dashboard.css'
```
Comment (shaunorpen) [inline src/components/DashboardHome/DashContent.js:5]: Do we need the second import of Dashboard.css?

File: src/components/DashboardHome/DashContent.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (shaunorpen) [inline src/components/DashboardHome/DashContent.js:1]: Are you using standard.js to format code? Everyone else (I think) is using no semi-colon at the end of lines. Can we update these files to follow that standard?

File: src/components/DashboardHome/Dashboard.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    position: fixed;
+}
+
+.dashDiv {
+    background-color: white;
+    width: 100vw;
+    height: 73vh;
+    border-radius: 25px 25px 0 0;
+    margin-top: 4vh;
+    /* position: sticky; */
```
Comment (shaunorpen) [inline src/components/DashboardHome/Dashboard.css:13]: Perhaps delete commented out code before merging?

**Review by aapetsi — APPROVED** [review]

File: src/components/DashboardHome/Footer.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (shaunorpen) [inline src/components/DashboardHome/Footer.js:1]: Do we need this component, or could we leverage the one Alex developed at inventory/nav.js? (Or use yours instead...I just don't want to see two components with the same code in our codebase :))

**Review by shaunorpen — CHANGES_REQUESTED** [review]
A few questions / observations before I'm happy to approve:).

File: src/components/DashboardHome/DashContent.js (hunk 1 above)
Comment (vunderkind) [reply src/components/DashboardHome/DashContent.js:5]: Sorry - my mistake. Resolved.

File: src/components/DashboardHome/Footer.js (hunk 4 above)
Comment (vunderkind) [reply src/components/DashboardHome/Footer.js:1]: Understood. I believe this is resolved as Alex says his footer version has a bug. Also his footer wasn't a standalone component, so I'm keeping mine. Hope this works!

Comment (shaunorpen) [reply src/components/DashboardHome/Footer.js:1]: Sure. Happy with that.

File: src/components/DashboardHome/DashContent.js (hunk 2 above)
Comment (vunderkind) [reply src/components/DashboardHome/DashContent.js:1]: Done!

**Review by shaunorpen — APPROVED** [review]
Nice work! Approved.

**Pull request merged**